src/main/java/ui/controller/MapController.java

```diff
@@ -38,6 +40,40 @@
 
 	private ArrayList<Line> currentPath = new ArrayList<Line>();
 
+	public class MapZoomHandler implements EventHandler<ScrollEvent> {
```

Comment:
Braces, but... ugh, whatever. @solashirai just make sure to change those settings in IntelliJ, okay?